CHANGELOG.md

```diff
@@ -2,14 +2,15 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
-## Wazuh v4.7.0 - OpenSearch Dashboards 2.6.0 - Revision 01
+## Wazuh v4.7.0 - OpenSearch Dashboards 2.6.0 - Revision 00
```

Comment:
**nitpick**: change to revision 01

Reply:
Solved in https://github.com/wazuh/wazuh-kibana-app/pull/5597/commits/8f2d5bf48f9dbeff9664a5744a455b43071f5116.